test-data/encode.broad.peak

```diff
@@ -0,0 +1,6 @@
+# TODO For the converter tests the header is commented fails otherwise
```

Comment:
Should this and the following files go to https://github.com/galaxyproject/galaxy-test-data (when you're done)?

Reply:
Done. Lets see if this works